### ISSUE TITLE
Search for persons with a members account

### ISSUE THREAD

**Issue by jefft** [tbar0970/jethro-pmm#879]
It would be useful to know which persons have signed up for the `/members` interface.

This can be seen for any particular user in their 'Accounts' tab:

![image](https://user-images.githubusercontent.com/205995/224595639-0a5cd777-62a3-47a7-9a10-1907d42f37e6.png)

but it would be nice to be able to search for such users, by having a **Has a Members Area account** field in the Persons report:
![image](https://user-images.githubusercontent.com/205995/224595974-cd28c219-7f83-4b70-a7ce-8b038ac6aaab.png)

**Comment by tbar0970** [comment]
Related: https://github.com/tbar0970/jethro-pmm/issues/53

Adding this feature to person reports would have the upside of being able to combine it with other filters (eg "show me all the men who have member accounts" ?!).  But the downside is adding complexity to an already-complex page.

We might want to see
- who has tried to create an account but not net completed the process
- who has successfully created an account
- who has logged in to the members area in the last [x] days

**Comment by jefft** [comment]
Yes, perhaps adding a 'Member-only Accounts' section to the current User Accounts admin page would be better.
